lib/mayaHydra/hydraExtensions/sceneIndex/CMakeLists.txt

```
@@ -32,6 +33,7 @@ set(HEADERS
     mayaHydraSceneIndexDataFactoriesSetup.h
     mayaHydraMayaFilteringSceneIndexData.h
     mayaHydraMayaFilteringSceneIndexDataConcreteFactory.h
+    mayaHydraMayaUsdProxyShapeSceneIndex.h
```

With our recent changes to the coding guidelines I believe these file names should be:
```suggestion
    mhMayaUsdProxyShapeSceneIndex.h
```